src/main/java/ch/njol/skript/test/runner/EffObjectives.java

```diff
@@ -0,0 +1,122 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.test.runner;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import com.google.common.collect.HashMultimap;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Multimap;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.NoDoc;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+
+@Name("Objectives")
+@Description("An effect to setup required objectives for JUnit tests to complete.")
+@NoDoc
+public class EffObjectives extends Effect  {
+
+	static {
+		if (TestMode.ENABLED)
+			Skript.registerEffect(EffObjectives.class,
+					"ensure [[junit] test] %string% completes [(objective|trigger)[s]] %strings%",
+					"complete [(objective|trigger)[s]] %strings% (for|on) [[junit] test] %string%"
+			);
+	}
+
+	private final static Multimap<String, String> requirements = HashMultimap.create();
+	private final static Multimap<String, String> completness = HashMultimap.create();
```

Comment:
```suggestion
	private static final Multimap<String, String> requirements = HashMultimap.create();
	private static final Multimap<String, String> completness = HashMultimap.create();
```
Also, should be `completeness` instead of `completness`